src/features/view-subdomains/SubdomainTableRow/SubdomainTableRow.tsx

```diff
@@ -65,25 +60,26 @@ export const SubdomainTableRow: FC<SubdomainTableRowProps> = ({
 						<span>0://{zna}</span>
 					</div>
 				</div>
-			</TableData>
-			<TableData alignment={'right'} className={styles.Metrics}>
+			</Cell>
+			<Cell alignment={'right'} className={styles.Metrics}>
 				<SkeletonText
 					asyncText={{
 						text: volume,
```

Comment:
Weird I thought I fixed these up...same as the card -

Whilst we're here could we add `errorText: '-',` below line 68: 

<img width="1028" alt="Screenshot 2023-03-01 at 18 34 30" src="https://user-images.githubusercontent.com/39112648/222073600-8cb8352c-2045-4a06-943a-5ade923beac4.png">